cmd/gobgp/lib/lib.go

```diff
@@ -1,78 +0,0 @@
-// Copyright (C) 2015 Nippon Telegraph and Telephone Corporation.
```

Comment:
These changes are not aligned with build.sh, gobgp_api_client.cc (sample). 

Still, the gobgp_api_client.cc sample file looking for "libgobgp.h" file and build.sh is pointing to the removed "lib" directory.  These files needs to be updated.